charts/rancher-k3s-upgrader/0.3.0/templates/clusterrolebinding.yaml

```diff
@@ -1,12 +1,12 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
 metadata:
-  name:  system-upgrade
+  name:  system-upgrade-controller
 roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: ClusterRole
   name: cluster-admin
 subjects:
-- kind: ServiceAccount
-  name: system-upgrade
-  namespace: cattle-system
+  - kind: ServiceAccount
+    name: system-upgrade-controller
+    namespace: cattle-system
```

Comment:
was the change in spacing/indent intentional? 

there is also a missing new line at the end of the file